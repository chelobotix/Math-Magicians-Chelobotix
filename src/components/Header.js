import NavBar from './NavBar';
import style from '../styles/Header.module.css';

const Header = () => (
  <div className={style.container}>
    <h1 className={style.logo}>Math Magicians</h1>
    <NavBar />
  </div>
);

export default Header;
