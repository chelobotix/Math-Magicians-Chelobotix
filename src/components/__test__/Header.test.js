import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Header from '../Header';

const MockHeader = () => (
  <BrowserRouter>
    <Header />
  </BrowserRouter>
);

describe('Header Component', () => {
  test('Tree', () => {
    const tree = renderer.create(<MockHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
