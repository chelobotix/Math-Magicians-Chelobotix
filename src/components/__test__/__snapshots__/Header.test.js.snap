// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Tree 1`] = `
<div
  className="container"
>
  <h1
    className="logo"
  >
    Math Magicians
  </h1>
  <nav
    className="nav"
  >
    <ul
      className="ul"
    >
      <li>
        <a
          aria-current="page"
          className="active link"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="liBorder"
      >
        <a
          className="noActive link"
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          className="noActive link"
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
